vendor/github.com/urfave/cli/help_minimal.go

```diff
@@ -0,0 +1,399 @@
+		return nil
+	}
+
+	if c.Command != nil {
+		return ShowCommandHelp(c, c.Command.Name)
+	}
+
+	return ShowCommandHelp(c, "")
+}
+
+// ShowVersion prints the version number of the App
+func ShowVersion(c *Context) {
+	VersionPrinter(c)
+}
+
+func printVersion(c *Context) {
+	_, _ = fmt.Fprintf(c.App.Writer, "%v version %v\n", c.App.Name, c.App.Version)
+}
+
+// ShowCompletions prints the lists of commands within a given context
+func ShowCompletions(c *Context) {
+	a := c.App
+	if a != nil && a.BashComplete != nil {
+		a.BashComplete(c)
+	}
+}
+
+// ShowCommandCompletions prints the custom completions for a given command
+func ShowCommandCompletions(ctx *Context, command string) {
+	c := ctx.App.Command(command)
+	if c != nil {
+		if c.BashComplete != nil {
+			c.BashComplete(ctx)
+		} else {
+			DefaultCompleteWithFlags(c)(ctx)
+		}
+	}
+
+}
+
+var appHelpTextFmt = `NAME:
+   %s - %s
+
+USAGE:
+   [global options] command [command options] [arguments...]
+
+VERSION:
+   %s
+
+DESCRIPTION:
+   %s
+
+COMMANDS:
+%s
+GLOBAL OPTIONS:
+%s
+`
+
+var cmdHelpTextFmt = `NAME:
+   %s - %s
+
+USAGE:
+   %s [command options] [options]
+
+OPTIONS:
+%s
+`
+
+// printHelpCustomNoTemplate is the default implementation of HelpPrinterCustom.
+//
+// Substitutes printHelpCustom from help.go removing templates support
+func printHelpCustomNoTemplate(out io.Writer, _ string, data interface{}, _ map[string]interface{}) {
+	helpStr := ""
+	switch d := data.(type) {
+	case *App:
+		commandsStr := ""
+		for _, cmd := range(d.Commands) {
+			commandsStr = commandsStr + fmt.Sprintf("   %s\t%s\n", cmd.Name, cmd.Usage)
+		}
+		optionsStr := ""
+		for _, flag := range(d.Flags) {
+			optionsStr = optionsStr + fmt.Sprintf("   %s\n", flag)
+		}
+		helpStr = fmt.Sprintf(appHelpTextFmt, d.Name, d.Usage, d.Version, d.Description, commandsStr, optionsStr)
+	case *Command:
+		optionsStr := ""
+		for _, flag := range(d.Flags) {
+			optionsStr = optionsStr + fmt.Sprintf("   %s\n", flag)
+		}
+		helpStr = fmt.Sprintf(cmdHelpTextFmt, d.Name, d.Usage, d.Name, optionsStr)
+	}
+
+	w := tabwriter.NewWriter(out, 1, 8, 2, ' ', 0)
+	w.Write([]byte(helpStr))
+	w.Flush()
+}
+
+func printHelp(out io.Writer, templ string, data interface{}) {
+	HelpPrinterCustom(out, templ, data, nil)
+}
+
+func checkVersion(c *Context) bool {
+	found := false
+	for _, name := range VersionFlag.Names() {
+		if c.Bool(name) {
+			found = true
+		}
+	}
+	return found
+}
+
+func checkHelp(c *Context) bool {
+	found := false
+	for _, name := range HelpFlag.Names() {
+		if c.Bool(name) {
+			found = true
+		}
+	}
+	return found
+}
+
+func checkCommandHelp(c *Context, name string) bool {
+	if c.Bool("h") || c.Bool("help") {
+		_ = ShowCommandHelp(c, name)
+		return true
+	}
+
+	return false
+}
+
+func checkSubcommandHelp(c *Context) bool {
+	if c.Bool("h") || c.Bool("help") {
+		_ = ShowSubcommandHelp(c)
+		return true
+	}
+
+	return false
+}
+
+func checkShellCompleteFlag(a *App, arguments []string) (bool, []string) {
```

Comment:
I don't believe we have completion enabled, and it's kind of awkward how you need to generate them in the first place, so I wouldn't bother...

Reply:
For this `help_minimal.go`, I replaced printHelpCustom with printHelpCustomNoTemplate but left the rest of the file intact. It does not have a real impact and I thought this would help maintenance, as a the git diffs will be easier to apply to both files.